### ISSUE TITLE
Can't read error message in catch function

### ISSUE THREAD

**Issue by riderx** [mongodb/stitch-js-sdk#103]
```
        client.login(username, password).then((userId) => {
            console.log('Successfully logged in as user', userId);
        })
            .catch((err) => {
                console.log('Error logging in with email/password auth:', err);
            });
```
Everithing work if the loggin is good but if password is wrong i get 
`Error logging in with email/password auth: Error`
not the reason 😢

**Comment by edaniels** [comment]
@riderx we're working on a fix for this now that will be in version 3.0.1

**Comment by riderx** [comment]
thank's a lot for that ! <3

**Comment by edaniels** [comment]
Hey @riderx , this should be fixed in 3.0.0 when it's released.
